tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclAssertCallBuilderTest.xtend

```diff
@@ -12,46 +12,50 @@ class TclAssertCallBuilderTest extends AbstractParserTest {
 	@Test
 	def void testEqualsGen() {
 		// given
-		val assertionTestStep = parseAssertionTestStep('''- assert variable == "test"''')
+		val assertionTestStep = parseAssertionTestStep('- assert variable == "test"')
 
 		// when
-		val assertMethod = assertCallBuilder.build(assertionTestStep.expression)
+		val genCode = assertCallBuilder.build(assertionTestStep.expression)
+		val assertMethod = genCode.split(System.lineSeparator).last
 
 		// then
-		assertMethod.assertEquals('org.junit.Assert.assertEquals(variable, "test");')
+		assertMethod.assertEquals('org.junit.Assert.assertEquals("test", variable);')
```

Comment:
The 3 lines above are redundant for each test. 1) parse 2) extract 3) compare.
If we would extract this to a helper method the tests would be more readable.